### ISSUE TITLE
[bug] learning rate is not changing

### ISSUE THREAD

**Issue by neouyghur** [davidsandberg/facenet#518]
Hi, in my experiment, I didn't use the learning_rate_schedule_file for changing the learning rate. And my learning rate is not changing.  I think there is a bug in your code. 

In /src/train_tripletloss.py file,  there is a train function. This train function set the learning rate to the default input learning rate value after each epoch like the following code. 
```
   if args.learning_rate>0.0:
        lr = args.learning_rate
  else:
     ....
```

**Comment by davidsandberg** [comment]
If you are not using a learning rate schedule file I assume you have set the learning_rate_decay_factor. If not the learning rate should not change. But if that is the case the parameter that you are looking at only sets the base learning rate, and it's the function tf.train.exponential_decay that decays that based on the number of training steps.
